esp/xslt/esdl2ecl.xslt

```diff
@@ -239,6 +240,13 @@ END;
 
 </xsl:template>
 
+<xsl:template name="output_ecl_stringtype">
+  <xsl:choose>
+    <xsl:when test="$utf8strings='yes'">UTF8</xsl:when>
```

Comment:
it looks like the rest of the xsl wraps text in <xsl:text disable-output-escaping="yes">, do we need to do that here?

Reply:
In this case I know nothing will be escaped.  The characters are all ascii.